vm/src/stdlib/socket.rs

```diff
@@ -449,6 +449,7 @@ fn extend_module_platform_specific(_vm: &VirtualMachine, module: PyObjectRef) ->
 fn extend_module_platform_specific(vm: &VirtualMachine, module: PyObjectRef) -> PyObjectRef {
     let ctx = &vm.ctx;
 
+    #[cfg(not(target_os = "redox"))]
```

Comment:
You might consider modifying line 443 somewhat more above. This function will then be enabled for `not unix or redox`.

Reply:
No, that's intentional. I'm assuming in the future that there will be unix-only parts of the module that *are* compatible with redox, so once those are added they can all stay in the `unix` `extend_module_platform_specific` fn like for [`os`](https://github.com/RustPython/RustPython/blob/0129bb5c7589cde5aedb5f416aad4830a89b8792/vm/src/stdlib/os.rs).

Reply:
Okay.